parts/k8s/windowskubeletfunc.ps1

```diff
@@ -165,22 +165,41 @@ New-InfraContainer {
         "1803" { 
             $imageList = docker images $defaultPauseImage --format "{{.Repository}}:{{.Tag}}"
             if (-not $imageList) {
-                docker pull $defaultPauseImage                 
+                Invoke-Executable -Executable "docker" -ArgList @("pull", "$defaultPauseImage") -Retries 5 -RetryDelaySeconds 30

```

Comment:
is there a known CSE execution global timeout that the extra 2.5 mins could affect?

Reply:
There is a 90 minute global timeout for CSE execution.
Adding a few 2.5 minute retries should be ok.